### ISSUE TITLE
ClistNicer:"Show status text" feature for better accessibility

### ISSUE THREAD

**Issue by georgehazan** [miranda-ng/miranda-ng#272]
**Reported by Piciok on 25 Feb 2014 21:22 UTC**
I have thought long about in what way we could solve this difficult issue of ClistBlind. I was reminded of a little trick, that once upon a time used to work in old MirandaIM 0.7 line etc. with Jaws that on ClistModern, when enabling the second and third line as well as disabling the skins engine made it read contact names, statuses and status messages. I have rechecked it now with current Clist Modern and NVDA - this doesn't work. Only contact names are read. I then have switched to Clist Nicer and contact names are read as well as status messages if present. This is amazing! My belie now is that if you added another line, or a feature of showing the status text such as "online, away etc". This way we would get the effect achieved by ClistBlind as well as have the long-awaited feature of selecting contacts. Do you think it would be hard to implement even as an experiment? ClistClassic is at present inaccessible due to the use of QT libraries but if accessibility libraries were added, that wouldn't be an issue as well.

**Comment by georgehazan** [comment]
**Modified by wishmaster on 13 May 2014 18:16 UTC**

**Comment by georgehazan** [comment]
**Comment by Piciok on 30 Nov 2014 23:11 UTC**
Some additions:
1. Currently NVDA which I'm using as my primary screenreader, recognises ClistNicer as a tree view, but a very unusual one where it's not possible to tell how many items are there in the view. I guess it's some unusual tree view.
2. Currently it is possible to select contacts where applicable E.G. the ignore list in options -- Contacts -- Ignore; but screenreaders don't recognise those checkboxes. This means that currently blind users can't tell whether a certain contact is selected or not.

**Comment by georgehazan** [comment]
**Modified by Piciok on 30 Nov 2014 23:26 UTC**
